src/Tinkoff.ISA.sln.DotSettings

```diff
@@ -0,0 +1,3 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
@mmikirtumov @lesley29 
I saw it earlier in @lesley29 PR, but we don't have this dictionary file now. We were agreed to delete it?

Reply:
@lesley29 I think that we considered to remove this file, if I am not mistaken :)

Reply:
Yep, we don't have this file in "scalable arch" branch https://github.com/TinkoffCreditSystems/ISA/tree/scalable_architecture/src

Reply:
ok, will remove it :-)